config_option_database/utils/BisonGraph.py

```diff
@@ -60,3 +60,33 @@ def make_terminal(self, node):
     
     def remove(self, node):
         self.graph.remove_node(node)
+
+    def get_paths(self, node='$accept', paths=None, stack=None):
+        if stack is None:
+            stack = []
+        if paths is None:
+            paths = [()]
+
+        if node in stack:
+            return paths
+        stack.append(node)
+
+        if self.is_junction(node):
+            for child in self.get_children(node):
+                if self.is_terminal(child):
+                    if child == '$end':
+                        break
+                    paths = paths[:]
```

Comment:
The function gets `paths` as an argument, and returns `paths`. Although the naming is unfortunate, it is intentional, to not change the `paths` in the argument of the function, because it would have an effect on other rules. Easiest is to comment it out, and run the `pytest`. But the copy should not be done for every child, so it is better to move it to the beginning of the function, so it only runs once.